api/dbadapters/bigquery.ts

```diff
@@ -1,11 +1,13 @@
 import { Credentials } from "@dataform/api/commands/credentials";
+import { adapters } from "@dataform/core";
```

Comment:
i think this is a bit sketchy. we want to be removing dependencies on `core` from `api`, not adding them. it looks like we only use this to determine the metadata table's name, do we really need to do that instead of hardcoding the table name?